trainer/forward_trainer.py

```diff
@@ -54,26 +54,25 @@ def train_session(self, model: ForwardTacotron,
         pitch_loss_avg = Averager()
         device = next(model.parameters()).device  # use same device as model parameters
         for e in range(1, epochs + 1):
-            for i, (x, m, ids, x_lens, mel_lens, dur, pitch) in enumerate(session.train_set, 1):
-
+            for i, batch in enumerate(session.train_set, 1):
+                batch = to_device(batch, device=device)
```

Comment:
imo this scope would be more readable with
`x, m, dur, x_lens, mel_lens, pitch = batch['x'], ....`